sphinx/builders/epub.py

```diff
@@ -187,7 +187,8 @@ class EpubBuilder(StandaloneHTMLBuilder):
     add_permalinks = False
     # don't add sidebar etc.
     embedded = True
-
+    # dont' create links to original images from images
+    override_post_process_images = False
```

Comment:
The name does not represent the behavior.

As you commented on, it controls to create links to an original, not resized images. What about `link_to_original_images = [True | False]`?